CefSharp/Internals/JavascriptObjectRepository.cs

```diff
@@ -738,7 +738,7 @@ private static bool IsComplexType(Type type)
                 baseType = Nullable.GetUnderlyingType(type);
             }
 
-            if (baseType == null || baseType.IsArray || baseType.Namespace.StartsWith("System"))
+            if (baseType == null || baseType.IsArray || (baseType.Namespace != null && baseType.Namespace.StartsWith("System")))
```

Comment:
Should be able to simplify this with a null conditional operator.

https://docs.microsoft.com/en-us/dotnet/csharp/language-reference/operators/member-access-operators#null-conditional-operators--and-

Reply:
For the time being, this will not work. Because the result of the null conditional operator is Nullable<bool>, it cannot satisfy the bool type constraint of if

Reply:
```suggestion
            if (baseType == null || baseType.IsArray || baseType.Namespace?.StartsWith("System") == true)
```

Reply:
I believe `(baseType?.Namespace.StartsWith("System") ?? false)` would also have worked though the `== true` syntax is nicer so going with that. @campersau Thanks 👍